ocean/nemo/nemolite2d/manual_versions/psykal_ocl/fortran/Makefile

```diff
@@ -0,0 +1,42 @@
+# This Makefile picks up the compiler to use, the location of the
+# OpenCL libraries and any flags from environment variables. e.g. to
+# use AMD's dispatch library:
+# 
+# export F90=gfortran
```

Comment:
Are these commented out compile options required?

Reply:
No, they're just to show a user what they need to set before running make.